### PR TITLE
Feature/klima 11 frontend api connection

### PR DESCRIPTION
**Pull request by tuj**
#### Link to ticket

https://jira.itkdev.dk/browse/KLIMA-11

#### Description

* Connects the frontend with the api.
* Fixed exposed tile entry animation.

Example:
https://aarhusmosaic.local.itkdev.dk/?id=1&key=123456789

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

### CODE REVIEW

File: assets/react/components/GridItem.jsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import React from "react";
 import styled from 'styled-components'
-import Icon from './Icons/Icons';
+import Icon from './icon';
 
-const GridItem = ({ variant, description, image, exposed, tileIcons, tileBorders }) => {
+const GridItem = ({ variant, description, image, exposed, showIcons, showBorders }) => {
```
Comment (yepzdk) [inline assets/react/components/GridItem.jsx]: Should we not keep the naming `tileIcons` etc. to better differentiate tile and expose?

File: src/DataFixtures/data/screen.json
Comment (yepzdk) [inline src/DataFixtures/data/screen.json:1]: These settings does not match those from settings.js: https://github.com/itk-dev/aarhusmosaic/blob/develop/assets/react/Data/settings.js

File: assets/react/controllers/Mosaic.jsx [hunk 2]
```diff
@@ -1,66 +1,159 @@
...
+
+    useEffect(() => {
+        if (screen === null) {
+            return;
+        }
+
+        const numberOfTiles = screen.gridColumns * screen.gridRows;
+
+        setConfig({
+            gridColumns: screen.gridColumns ?? 6,
```
Comment (yepzdk) [inline assets/react/controllers/Mosaic.jsx:101]: Do we need fallback for gridColumns, both here and in GlobalStyles.js?

**Review by yepzdk — COMMENTED** [review]
Some issues around settings/config should be addressed.
Also when setting "showIcons":false in admin removes all icons, aslo in expose.

File: assets/react/components/GridItem.jsx (hunk 1 above)
Comment (tuj) [reply assets/react/components/GridItem.jsx]: done

File: src/DataFixtures/data/screen.json
Comment (tuj) [reply src/DataFixtures/data/screen.json:1]: There was not consistent naming in the different files, so I just chose some names :)

File: assets/react/controllers/Mosaic.jsx (hunk 2 above)
Comment (tuj) [reply assets/react/controllers/Mosaic.jsx:101]: Removed in global styles when we know they are initialized in config.

**Review by yepzdk — APPROVED** [review]
Very good work @tuj

**Pull request merged**